components/footer.jsx

```diff
@@ -98,16 +100,14 @@ var Footer = React.createClass({
               <ul className="list-unstyled">
                 <li><a href="https://webmaker.org/">Webmaker</a></li>
                 <li><a href="https://sendto.mozilla.org/">Donate</a></li>
-                <li><a href="">Legal</a></li>
-                <li><a href="">Privacy</a></li>
+                <li><a href="">Twitter</a></li>
               </ul>
             </div>
             <div className="col-xs-6">
               <ul className="list-unstyled">
-                <li><a href="">Contact</a></li>
-                <li><a href="">Twitter</a></li>
-                <li><a href="">Facebook</a></li>
-                <li><a href="">Partners</a></li>
+                <li><a href="">Legal</a></li>
```

Comment:
Actually, wait...it's easier just to link to the Mozilla Legal and Privacy pages, isn't it? Let's do that. They're even more comprehensive, so it's perhaps a better choice all around.